dataactvalidator/config/sqlrules/c9_award_financial.sql

```diff
@@ -1,14 +1,14 @@
 SELECT afa.row_number, afa.fain, afa.uri
 FROM award_financial_assistance AS afa
 WHERE afa.submission_id = {0}
-	AND afa.federal_action_obligation > 0
+	AND CAST(afa.federal_action_obligation as numeric) > 0
```

Comment:
So part of the motivation here is that the break will occur during the cross-file validation where there is error reporting to the user, rather than happening on D2 file validation which just causes a loop back to the beginning of the process without reporting anything to the user.  That said, the Smartronix system is supposed to be validating D2 for this type of thing, but if they do send data of the wrong type this will at least have the error occur on a step that is visible to the frontend.